OpenRA.Mods.Common/Widgets/WorldCommandWidget.cs

```diff
@@ -191,15 +191,24 @@ bool CycleBases()
 				.Select(b => b.Actor)
 				.ToList();
 
-			if (!bases.Any())
-				return true;
+            // If no base is found all existing buildings are selected.
+            if (!bases.Any())
+            {
+				bases = world.ActorsWithTrait<TargetableBuilding>()
+                .Where(a => a.Actor.Owner == world.LocalPlayer)
+                .Select(b => b.Actor)
+                .ToList();
+                if (!bases.Any())
+					return true;
+            }   
```

Comment:
Just in case you didn't understand that - OpenRA's Utility has a `make check` function, that checks the code for style errors and warns you.
It is what our CI system (Travis) is using, and this is why it's marked the build as failing below.